test/e2e/testdata2/guestbook-partial-symlink-files/guestbook-ui-deployment.yaml

```diff
@@ -0,0 +1 @@
+../guestbook/guestbook-ui-deployment.yaml
```

Comment:
I have to put all test files related to symlinks to a dedicated folder `testdata2` otherwise it will fail other tests with `repository contains out-of-bounds symlinks` errors. I think that is because in the codebase we deny all absolute symlinks (https://github.com/argoproj/argo-cd/blob/master/util/app/path/path.go#L67), however, in our tests, we use local repo in `/tmp/` folder and that means all files are absolute paths.